test/plugins/individual-collateral/cbeth/CBETHCollateral.test.ts

```diff
@@ -0,0 +1,226 @@
+const reduceRefPerTok = async (
+  ctx: CbEthCollateralFixtureContext,
+  pctDecrease: BigNumberish
+) => {
+  await whileImpersonating(hre, CB_ETH_ORACLE, async oracleSigner => {
+    const rate = await ctx.cbETH.exchangeRate()
+    await ctx.cbETH.connect(oracleSigner).updateExchangeRate(
+      rate.sub(rate.mul(bn(pctDecrease)).div(bn('100')))
+    )
+  })
+  const lastRound = await ctx.refPerTokChainlinkFeed.latestRoundData()
+  const nextAnswer = lastRound.answer.sub(lastRound.answer.mul(pctDecrease).div(100))
+  await ctx.refPerTokChainlinkFeed.updateAnswer(nextAnswer)
+}
+
+// prettier-ignore
+const increaseRefPerTok = async (
+  ctx: CbEthCollateralFixtureContext,
+  pctIncrease: BigNumberish
+) => {
+  await whileImpersonating(hre, CB_ETH_ORACLE, async oracleSigner => {
+    const rate = await ctx.cbETH.exchangeRate()
+    await ctx.cbETH.connect(oracleSigner).updateExchangeRate(
+      rate.add(rate.mul(bn(pctIncrease)).div(bn('100')))
+    )
+  })
+  const lastRound = await ctx.refPerTokChainlinkFeed.latestRoundData()
+  const nextAnswer = lastRound.answer.add(lastRound.answer.mul(pctIncrease).div(100))
+  await ctx.refPerTokChainlinkFeed.updateAnswer(nextAnswer)
+}
+const getExpectedPrice = async (ctx: CbEthCollateralFixtureContext): Promise<BigNumber> => {
+  const clData = await ctx.chainlinkFeed.latestRoundData()
+  const clDecimals = await ctx.chainlinkFeed.decimals()
+
+  const clRptData = await ctx.refPerTokChainlinkFeed.latestRoundData()
+  const clRptDecimals = await ctx.refPerTokChainlinkFeed.decimals()
+
+  return clData.answer
+    .mul(bn(10).pow(18 - clDecimals))
+    .mul(clRptData.answer.mul(bn(10).pow(18 - clRptDecimals)))
+    .div(fp('1'))
+}
+
+/*
+  Define collateral-specific tests
+*/
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const collateralSpecificConstructorTests = () => {}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const collateralSpecificStatusTests = () => {}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const beforeEachRewardsTest = async () => {}
+
+export const defaultRethCollateralOpts: CollateralOpts = {
+  erc20: CB_ETH,
+  targetName: ethers.utils.formatBytes32String('ETH'),
+  rewardERC20: WETH,
+  priceTimeout: PRICE_TIMEOUT,
+  chainlinkFeed: ETH_USD_PRICE_FEED,
+  oracleTimeout: ORACLE_TIMEOUT,
+  oracleError: ORACLE_ERROR,
+  maxTradeVolume: MAX_TRADE_VOL,
+  defaultThreshold: DEFAULT_THRESHOLD,
+  delayUntilDefault: DELAY_UNTIL_DEFAULT,
+  revenueHiding: fp('0'),
+}
+
+/*
+  Run the test suite
+*/
+
+const opts = {
+  deployCollateral,
+  collateralSpecificConstructorTests,
+  collateralSpecificStatusTests,
+  beforeEachRewardsTest,
+  makeCollateralFixtureContext,
+  mintCollateralTo,
+  reduceTargetPerRef,
+  increaseTargetPerRef,
+  reduceRefPerTok,
+  increaseRefPerTok,
+  getExpectedPrice,
+  itClaimsRewards: it.skip,
+  itChecksTargetPerRefDefault: it.skip,
+  itChecksRefPerTokDefault: it.skip,
```

Comment:
I think both `itChecksRefPerTokDefault` and `itHasRevenueHiding` should be On (ie `it`).

Reply:
Let's make sure the tests pass once we enable these